src/main/java/com/github/sgov/server/util/Utils.java

```diff
@@ -29,18 +30,32 @@ public static VocabularyFolder getVocabularyFolder(final File root,
     /**
      * Creates a new attachment folder for the given attachment.
      *
-     * @param root          Root folder for attachments
-     * @param iri IRI of the attachment
+     * @param root Root folder for attachments
+     * @param iri  IRI of the attachment
      * @return folder for the given attachment
      */
     public static AttachmentFolder getAttachmentFolder(final File root,
                                                        final String iri) {
         return new AttachmentFolder(
             Paths.get(root.getAbsolutePath() + "/"
-                   + (CONTENT_ROOT + "/attachments/" + getAttachmentId(iri)))
+                    + (CONTENT_ROOT + "/attachments/" + getAttachmentId(iri)))
                 .toFile());
     }
 
+    /**
+     * Returns a relative path to folder from a given root folder.
+     *
+     * @param folder given folder
+     * @param root   root folder
+     * @return relative path to the given vocabulary
+     */
+    public static Path getRelativePathToFolder(final File root,
+                                               final Folder folder) {
+        String folderPath = folder.getFolder().getAbsolutePath();
+        String relativePath = folderPath.replace(root.getAbsolutePath() + "/", "");
+        return Paths.get(relativePath);
```

Comment:
@Mighantos sorry, in addition i realized that this might not work that well. I would use API of Path if possible, see e.g. :
https://stackoverflow.com/questions/204784/how-to-construct-a-relative-path-in-java-from-two-absolute-paths-or-urls
```
        Path pathAbsolute = Paths.get("/var/data/stuff/xyz.dat");
        Path pathBase = Paths.get("/var/data");
        Path pathRelative = pathBase.relativize(pathAbsolute);
```

Your solution might have problems with symlinks i believe.

Reply:
Implemented in commit [Change to using Path.relativize](https://github.com/opendata-mvcr/sgov/pull/153/commits/a0476ae9fdfed458ceaae6be898e728c91fac6ae)